app/src/main/java/org/zephyrsoft/trackworktime/model/WeekState.java

```diff
@@ -0,0 +1,32 @@
+package org.zephyrsoft.trackworktime.model;
+
+import androidx.annotation.NonNull;
+
+public class WeekState {
+
+	@NonNull public WeekRowState header, monday, tuesday, wednesday, thursday, friday, saturday, sunday,
+			totals;
+	{
+		header = new WeekRowState();
+		monday = new WeekRowState();
+		tuesday = new WeekRowState();
+		wednesday = new WeekRowState();
+		thursday = new WeekRowState();
+		friday = new WeekRowState();
+		saturday = new WeekRowState();
+		sunday = new WeekRowState();
+		totals = new WeekRowState();
+	}
```

Comment:
See my comment on WeekRowState - here it's even nearly the same amount of lines either way...

Reply:
I agree it's a bit weird in previous example, but even weirder here 😄 My goal was to keep `WeekRowState` immutable, but had to change them to this mess, otherwise it would end up similar as above. Will change it.

Reply:
Updated. Will add get/setters at later date. Due to how `WeekStateCalculator` is currently designed, I would basically need to mutate `WeekState` values using getters, so I'd rather postpone them for now.